lib/cli.js

```diff
@@ -39,7 +39,7 @@ yargs
         },
     })
     .command({
-        command: 'deploy [BU] [TYPE] [KEY] [--fromRetrieve] [--refresh]',
+        command: 'deploy [BU] [TYPE] [KEY] [--fromRetrieve] [--refresh] [--execute]',
```

Comment:
- [ ] lets remove that here and add it to the new group() method for the deploy command :)

Reply:
requires that you merge develop into this branch first